compiler/frontend/src/org/jetbrains/kotlin/diagnostics/PositioningStrategies.kt

```diff
@@ -462,6 +462,17 @@ object PositioningStrategies {
         }
     }
 
+    /**
+     * Mark the name of a named argument. If the given element is not a named argument or doesn't have a name, then the entire given element
+     * is marked instead.
+     */
+    @JvmField
+    val NAME_OF_NAMED_ARGUMENT: PositioningStrategy<PsiElement> = object : PositioningStrategy<PsiElement>() {
```

Comment:
It would work and this way we will need to mark `NAMED_ARGUMENTS_NOT_ALLOWED` to be reported on `KtValueArgument` instead of `PsiElement`, which difffers from FE1.0. But I don't see a problem of doing this since our new checkers report on the whole argument rather than the name reference. Let me do it.